source/yats/include/yats/task_helper.h

```diff
@@ -57,16 +71,16 @@ struct task_helper
     using output_type = Return;
 
     using input_tuple = std::tuple<ParameterTypes...>;
-    using output_tuple = typename output_wrapper<output_type>::tuple;
+    using output_tuple = typename output_wrapper<Return>::tuple;
```

Comment:
Why did u change this to Return? I think both is fine, just curious

Reply:
I wanted to change output_type but I found an implementation where this was not necessary. Removing one type indirection is still not a bad idea.